pkg/proxy/winkernel/proxier.go

```diff
@@ -354,7 +354,7 @@ func newSourceVIP(hns HostNetworkService, network string, ip string, mac string,
 
 func (ep *endpointsInfo) Cleanup() {
 	Log(ep, "Endpoint Cleanup", 3)
-	if ep.refCount != nil {
+	if !ep.GetIsLocal() && ep.refCount != nil {
```

Comment:
just double-checking: we do not want to decrement the endpoint ref count if the endpoint is local?